docs/HyBIG-Example-Usage.ipynb

```diff
@@ -453,21 +599,55 @@
     "    for file_future in harmony_client.download_all(\n",
     "        tiled_job_id, overwrite=True, directory=example5_output_dir\n",
     "    )\n",
-    "]"
+    "]\n",
+    "rename_files(downloaded_tiled_outputs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8d1a6948",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The tiles have expected dimensions 4096 x 4096:\n",
+    "!file hybig-output/example5/VCF5KYR_1991001_001_2018224205008.r00c00.png"
```

Comment:
I think this needs updating to account for the new file naming. The saved equivalent I have is: "1991001_001_2018224205008.r00c00.png".

Reply:
Got you. I was using `harmony-py==0.4.10`, which is compatible with `docs/requirements.txt`. I tried things again with `harmony-py==0.4.14` (latest) and it all worked nicely.

It's probably worth bumping up that version listed in the requirements file to one that will definitely produce the right file paths. I'd just plump for `harmony-py~=0.4.14` if that sounds good?

Reply:
I thought it was already 0.4.14. That why I was confused.